### DISCUSSION TITLE
[SN Utils] CTRL or CMD-Click to open source

### DISCUSSION THREAD

**Discussion by Stefke232** [arnoudkooi/SN-Utils#350] [Q&A]
Hello,

Is there an option to disable ctrl-click to open source report?

On the dashboard page, I need to select multiple fields in the table while holding down ctrl and I just get a new tab opened that needs to be closed.

**Comment by arnoudkooi** [comment]
Do you have details, screenshots I can use for testing? Idea is that it should only work like this in the widget header

**Reply by Stefke232** [reply]
It doesn't work like that at all.

For reports that are made by someone else and shared with me, it will not open the source report. Even if I control-click on the banner of the widget.
For the reports that I created and have full control and I am able to change the report, control-click will open source report when I click on field in the table. like in the image where priority is selected. 
Every time I select something to change in the table I get a new tab.

![image](https://user-images.githubusercontent.com/114781278/214620657-80f11373-7f60-4081-be40-47160de47405.png)

**Comment by arnoudkooi** [comment]
Sorry, forgot to look at this one...
I will add a condition in the next update to only listen when the header bar is clicked and will post when it is published.

**Comment by arnoudkooi** [comment]
Please try if it is fixed in 6.3.5.8, should only work in the widget header.
